common/theme.c

```diff
@@ -6,42 +6,112 @@
 #include "hbmenu_logo_light_bin.h"
 #include "hbmenu_logo_dark_bin.h"
 
+bool colorFromSetting(config_setting_t *rgba, color_t *col) {
+    if(rgba == NULL)
+        return false;
+    *col = MakeColor(config_setting_get_int_elem(rgba, 0), config_setting_get_int_elem(rgba, 1), config_setting_get_int_elem(rgba, 2), config_setting_get_int_elem(rgba, 3));
+    return true;
+}
+
 void themeStartup(ThemePreset preset) {
+    theme_t themeLight = (theme_t) { 
+        .textColor = MakeColor(0, 0, 0, 255),
+        .frontWaveColor = MakeColor(100, 212, 250, 255),
+        .middleWaveColor = MakeColor(100, 153, 255, 255),
+        .backWaveColor = MakeColor(154, 171, 255, 255),
+        .backgroundColor = MakeColor(233, 236, 241, 255),
+        .highlightColor = MakeColor(91, 237, 224, 255),
+        .separatorColor = MakeColor(219, 218, 219, 255),
+        .enableWaveBlending = 0,
+        .buttonAText = "\uE0E0",
+        .buttonBText = "\uE0E1",
+        //.buttonAImage = button_a_light_bin,
+        //.buttonBImage = button_b_light_bin,
+        .hbmenuLogoImage = hbmenu_logo_light_bin
+    };
+    
+    theme_t themeDark = (theme_t) { 
+        .textColor = MakeColor(255, 255, 255, 255),
+        .frontWaveColor = MakeColor(96, 204, 204, 255),
+        .middleWaveColor = MakeColor(66, 154, 159, 255),
+        .backWaveColor = MakeColor(73, 103, 169, 255),
+        .backgroundColor = MakeColor(45, 45, 50, 255),
+        .highlightColor = MakeColor(91, 237, 224, 255),
+        .separatorColor = MakeColor(219, 218, 219, 255),
+        .enableWaveBlending = 0,
+        .buttonAText = "\uE0A0",
+        .buttonBText = "\uE0A1",
+        //.buttonAImage = button_a_dark_bin,
+        //.buttonBImage = button_b_dark_bin,
+        .hbmenuLogoImage = hbmenu_logo_dark_bin
+    };
+    
+    theme_t *themeDefault;
+    config_t *cfg = NULL;
```

Comment:
Store on stack. "config_t cfg = {0};"

Cleanup once finished using it via config_destory is also missing.